src/vm/object.inl

```diff
@@ -257,7 +257,7 @@ inline /* static */ unsigned ArrayBase::GetBoundsOffset(MethodTable* pMT)
     if (!pMT->IsMultiDimArray()) 
         return(offsetof(ArrayBase, m_NumComponents));
     _ASSERTE(pMT->GetInternalCorElementType() == ELEMENT_TYPE_ARRAY);
```

Comment:
You can also fix `offsetof(ArrayBase, m_NumComponents)` a few lines above while you are on it. Replace it with `OBJECT_SIZE /* offset(ArrayBase, m_NumComponents */` ?

Reply:
Done. Thanks for catching this